app/views/add-reduction.html

```diff
@@ -0,0 +1,156 @@
+{% extends "includes/layout.html" %} {% block content %}
+
+<main id="content" role="main">
+  {% include "includes/breadcrumbs.html" %} 
+  
+  <header class="heading-xlarge" style="margin-top:0; margin-bottom:0">
+    New reduction
+  </header>
+  {%if failureText %}
+  <p class="message message--failure">{{ failureText }}</p>
+  {% endif %}
+  <p class="lede">Add a new reduction for <span class="bold">{{ title }}</span></p>
+  {% set first = true %}
+  <form action="/offender-manager/{{ linkId }}/add-reduction" method=POST id="reductionForm">
+    <div class="form-group" id="reduction_reason_selection">
+      <label class="form-label-bold" for="select-box">Reason for reduction</label>
+      <select class="form-control" id="select-box" name="reductionType" onChange="populateHours(), calculateExpiry()">        
+        <option value="" disabled selected>Please select a reason...</option>
+        {% for reason in referenceData %}
+        <option value="{{ reason.id }}">{{ reason.reasonShortName }}</option>
+        {% else %}
+        <option value="{{ reason.id }}">{{ reason.reasonShortName }}</option>
+        {% endfor %}
+      </select>
+    </div>
+
+    <div class="form-group" id="reduction_hours">
+      <label class="form-label-bold" for="hours">Hours</label>
+      <input class="form-control" id="hours" name="hours" type="number" style="width: 100px">
+    </div>
+
+    <div class="form-group" id="reduction_start_date">
+      <fieldset>
+        <legend>
+          <span class="form-label-bold">Reduction start date</span>
+          <span class="form-hint">For example, 31 3 1980</span>
+        </legend>
+        <div class="form-date" id="reduction_start_day">
+          <div class="form-group form-group-day">
+            <label class="form-label" for="start-day">Day</label>
+            <input class="form-control" id="start-day" name="red_start_day" type="number" min="1" max="31">
+          </div>
+          <div class="form-group form-group-month" id="reduction_start_month">
+            <label class="form-label" for="start-month">Month</label>
+            <input class="form-control" id="start-month" name="red_start_month" type="number" min="1" max="12">
+          </div>
+          <div class="form-group form-group-year" id="reduction_start_year">
+            <label class="form-label" for="start-year">Year</label>
+            <input class="form-control" id="start-year" name="red_start_year" type="number" min="0" max="2099" onchange="calculateExpiry()">
```

Comment:
Min? Sensible year? Should they be able to add reductions in the past?

Reply:
Cool, We can do that automagically. Good catch 👍